Makefile

```diff
@@ -133,18 +133,15 @@ aw-qt/media/logo/logo.icns:
 	mv build/MyIcon.icns aw-qt/media/logo/logo.icns
 
 dist/ActivityWatch.app: aw-qt/media/logo/logo.icns
-	pyinstaller --clean --noconfirm --windowed --codesign-identity $$APPLE_TEAMID aw.spec
-
-dist/notarize:
-	./scripts/notarize.sh
+	pyinstaller --clean --noconfirm --windowed aw.spec
 
 dist/ActivityWatch.dmg: dist/ActivityWatch.app
 	pip install dmgbuild
 	dmgbuild -s scripts/package/dmgbuild-settings.py -D app=dist/ActivityWatch.app "ActivityWatch" dist/ActivityWatch.dmg
-# Don't try to run this outside CI, it causes messes in your macos keychain
-codesign-dmg: dist/ActivityWatch.dmg
-	./scripts/ci/import-macos-p12.sh
-	codesign --verbose -s ActivityWatch --deep dist/ActivityWatch.dmg
+	codesign --verbose -s ${APPLE_PERSONAL_ID} dist/ActivityWatch.dmg
```

Comment:
Also, I might later wrap this in a:

```
ifdef SIGN
	...
endif
```

Such that it is turned off by default, and enabled by passing a env var `SIGN=true` (since it requires the keychain to be configured).

Reply:
That's one option, another option is to sign by default but in adhoc mode.
I believe that if you pass "-" it will sign `adhoc` with a default dev certificate installed on the machine.